### ISSUE TITLE
Conflicts between epub:type and role

### ISSUE THREAD

**Issue by mattgarrish** [w3c/epub-specs#714]
From Daniel Glazman:

If a given element holds both @role and @epub:type and they're conflicting, what happens? On the editing side, I don't know how to handle such a case. On the reading side, what should be the behaviour of a reading system? This must be specified.

Not a critical issue in 3.1 anymore, but we may want to look at guidance in the accessibility techniques.

**Comment by mattgarrish** [comment]
The accessibility specification has a section now about authoring both for their target uses. There's nothing to say about reading system behaviour as role isn't defined for any specific purpose in epub anymore.
